packages/playground/src/components/node_details_cards/card_details.vue

```diff
@@ -40,6 +40,7 @@
 
             <v-tooltip class="overflowText" v-if="item.hint && !item.icon" location="top" :text="item.hint">
               <template #activator="{ props }">
+                <span class="flag-avatar" v-if="item.name === 'Flag' && !item.imgSrc">NA</span>
```

Comment:
Would it be better if we added some padding top to the span?

Reply:
padding is added in flag-avatar class